Mathlib/MeasureTheory/Measure/LevyProkhorovMetric.lean

```diff
@@ -395,13 +444,188 @@ lemma continuous_levyProkhorov_to_probabilityMeasure :
                ge_iff_le, forall_exists_index]
     refine ⟨0, fun a i hia ↦ le_trans (integral_nonneg f_nn) (hia i rfl.le)⟩
 
-/-- The topology of the Lévy-Prokhorov metric is finer than the topology of convergence in
+/-- The topology of the Lévy-Prokhorov metric is at least as fine as the topology of convergence in
 distribution. -/
 theorem levyProkhorov_le_convergenceInDistribution :
     TopologicalSpace.coinduced (LevyProkhorov.probabilityMeasure (Ω := Ω)) inferInstance
       ≤ (inferInstance : TopologicalSpace (ProbabilityMeasure Ω)) :=
   (continuous_levyProkhorov_to_probabilityMeasure).coinduced_le
 
-end Levy_Prokhorov_comparison
+end Levy_Prokhorov_is_finer
+
+
+
+section Levy_Prokhorov_metrizes_convergence_in_distribution
+
+/-! ### On separable spaces the Lévy-Prokhorov distance metrizes convergence in distribution -/
+
+open BoundedContinuousFunction TopologicalSpace
+
+variable {ι : Type*} {Ω : Type*} [MeasurableSpace Ω]
+
+/-- In a separable pseudometric space, for any ε > 0 there exists a countable collection of
+disjoint Borel measurable subsets of diameter at most ε that cover the whole space. -/
+lemma SeparableSpace.exists_measurable_partition_diam_le
```

Comment:
It is not clear to me what is the optimal location for this lemma. It mixes topology and measure theory. `#find_home` suggests this file (although I find that conclusion slightly hard to believe).

I'd expect that the file `Mathlib.MeasureTheory.Constructions.BorelSpace.Basic` on Borel sigma algebras and `OpensMeasurableSpaces` should have approximately the right imports of topology and measure theory. However, this lemma is probably not core content there even if the necessary imports are in place there.

For the moment, the PR suggests to add the lemma here, but I am happy to hear better suggestions. Another natural option might be to create a new file about Borel spaces with separability assumption in the folder `Mathlib.MeasureTheory.Constructions.BorelSpace`. I'm happy with either.

Reply:
For now I agree it makes sense to keep it here.